docs/05_Mock_tests.md

```diff
@@ -0,0 +1,57 @@
+# Mock Tests
+
+A web crawler is by nature dependant on the external service which it is crawling.
+In order to have a reliable CI pipeline, this external dependency needs to be removed.
+For this there is a system in place to automatically save the requests made to a service and their responses.
+These can then be used in the CI pipeline to reliably test changes made to the Extractor and not have test failures due to API changes on the side of the service.
+
+## Multiple downloader implementations
+
+There are multiple implementations of the abstract class `Downloader`
+
+1. `DownloaderTestImpl` is used for running the test against the actual service.
+2. `RecordingDownloader` is used to save the request and response to the disk, thus creating the mock.
```

Comment:
Nothing wrong here, just thinking it would be better to use hardware storage-agnostic terms.

```suggestion
2. `RecordingDownloader` is used to save the request and response to a file, thus creating the mock.
```